### PR TITLE
[WIP] Add information on Swarm operations roles

### PR DESCRIPTION
**Pull request by traci-morrison**
### Proposed changes
- Add Swarm operations roles section
- Update H1 headers
- Moved the important rules regarding roles note to a more visible place (above the fold)

### Related issues (optional)
https://docker.atlassian.net/browse/ENGDOCS-144

### CODE REVIEW

**Comment by traci-morrison** [comment]
@DawnWood-Docker PTAL. This is the PR for the peer review document. Thanks!

@RomainBelorgey PTAL.

File: ee/ucp/authorization/define-roles.md [hunk 1]
```diff
@@ -34,20 +40,137 @@ the same name to different resource sets.
...
+
+```
+docker create [OPTIONS] IMAGE [COMMAND] [ARG...]
+```
+
+See [extended description](https://docs.docker.com/engine/reference/commandline/create/#extended-description) and [examples](https://docs.docker.com/engine/reference/commandline/create/#examples) for more information.
+
+#### Manage images
+
+Use the `docker image` command to mange images. Use the `docker image remove` command to remove one or more images.
```
Comment (RomainBelorgey) [inline ee/ucp/authorization/define-roles.md]: Manage and not mange :)

**Comment by RomainBelorgey** [comment]
Just a little typo, looks good :+1:

File: ee/ucp/authorization/define-roles.md [hunk 2]
```diff
@@ -34,20 +40,137 @@ the same name to different resource sets.
 1. Click **Roles** under **User Management**.
 2. Click **Create Role**.
 3. Input the role name on the **Details** page.
```
Comment (usha-mandya) [inline ee/ucp/authorization/define-roles.md]: s/Input/Enter

**Comment by usha-mandya** [comment]
Looks good Traci. Wondering whether the entries in the `Swarm operations roles` section can be represented in a table format to enhance readability? We could use three columns - Operation, Command, and Description. Just a suggestion.

**Comment by traci-morrison** [comment]
> Looks good Traci. Wondering whether the entries in the `Swarm operations roles` section can be represented in a table format to enhance readability? We could use three columns - Operation, Command, and Description. Just a suggestion.

Thanks @usha-mandya. I like the table suggestion. I moved that information into a table format. Can you do another review? Thanks!

**Pull request merged**